custom_components/tuya_local/devices/cecotec_readywarm_8400.yaml

```diff
@@ -0,0 +1,103 @@
+name: Bladeless Heater
+products:
+  - id: var0f7dheclxpjuz
+    name: ReadyWarm Bladeless
+primary_entity:
+  entity: climate
+  translation_only_key: heater
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: preset_mode
+          conditions:
+            - dps_val: "null"
+              value: fan_only
+              hidden: true
+            - dps_val: "off"
+              value: fan_only
+            - dps_val: "low"
+              value: heat
+            - dps_val: "high"
+              value: heat
+            - dps_val: "auto"
+              value: heat
+    - id: 2
+      type: integer
+      name: temperature
+      unit: C
+      range:
+        min: 15
+        max: 40
+    - id: 3
+      type: integer
+      name: current_temperature
+      readonly: true
+    - id: 4
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: "off"
+          value: none
+          hidden: true
+        - dps_val: "low"
+          value: eco
+        - dps_val: "high"
+          value: boost
+        - dps_val: "auto"
+          value: auto
```

Comment:
climate does not have an "auto" preset predefined - maybe "comfort" is suitable (since the auto is most likely adjusting the heat based on how close the current temperature is to the target temperature, but if it is based on a sensor, maybe "activity" is better).